oneview/resource_firmware_drivers.go

```diff
@@ -0,0 +1,403 @@
+							Computed: true,
+						},
+						"release_data": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"resource_id": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+					},
+				},
+			},
+			"hpsum_version": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"iso_file_name": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"last_task_uri": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"mirror_list": {
+				Computed: true,
+				Type:     schema.TypeMap,
+				Elem: &schema.Schema{
+					Type: schema.TypeList,
+				},
+			},
+			"locations": {
+				Computed: true,
+				Type:     schema.TypeMap,
+				Elem: &schema.Schema{
+					Type: schema.TypeString,
+				},
+			},
+			"parent_bundle": {
+				Computed: true,
+				Type:     schema.TypeSet,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"parent_bundle_name": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"release_data": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"version": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+					},
+				},
+			},
+			"release_data": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"resource_id": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"resource_state": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"scope_uri": {
+				Computed: true,
+				Type:     schema.TypeString,
+			},
+			"signature_file_name": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"signature_file_required": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"supported_languages": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"supported_os_list": {
+				Computed: true,
+				Type:     schema.TypeSet,
+				Elem: &schema.Schema{
+					Type: schema.TypeString,
+				},
+				Set: schema.HashString,
+			},
+			"sw_packages_full_path": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"uuid": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"version": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"xml_key_name": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"baseline_uri": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"hotfix_uris": {
+				Type: schema.TypeSet,
+				Elem: &schema.Schema{
+					Type: schema.TypeString,
+				},
+				Set:      schema.HashString,
+				Required: true,
+				ForceNew: true,
+			},
+			"custom_baseline_name": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"initial_scope_uris": {
+				Optional: true,
+				Type:     schema.TypeSet,
+				Elem: &schema.Schema{
+					Type: schema.TypeString,
+				},
+				Set:      schema.HashString,
+				ForceNew: true,
```

Comment:
why have you made this attribute to forceNew?

Reply:
Because these four attributes will not present in the data_source file....this attributes is for create function.